src/main/java/net/coderbot/iris/Iris.java

```diff
@@ -63,7 +64,7 @@ public void onInitializeClient() {
 					// A lot of people are reporting visual bugs with Iris + Sodium. This makes it so that if we don't have
 					// the right fork of Sodium, it will just crash.
 					if (!versionString.startsWith("0.2.0+IRIS3")) {
-						throw new IllegalStateException("You do not have a compatible version of Sodium installed! You have " + versionString + " but 0.2.0+IRIS3 is expected");
+						sodiumInvalid = true;
```

Comment:
We probably still want to have some sanity check that crashes the game if the wrong version of Sodium is installed and the user tries to load a world.

Reply:
The user is never able to load the world with this implementation, clicking back closes the game. (I can't change what back says without creating a new Screen instance)

Reply:
> I can't change what back says without creating a new Screen instance

Is there a way to display it without the button though? That would reduce the confusion

Reply:
Not without a new screen, I could create one if there was demand but it would just need another class.

Reply:
> The user is never able to load the world with this implementation

Even though I too believe this to be true, it wouldn't hurt to still add a check & crash in world rendering to be 100% absolutely sure that nothing unexpected happens causing the game to proceed past the warning screen.

Reply:
Will do this later.